packages/react-17/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-17",
+  "private": "true",
+  "version": "0.0.0",
```

Comment:
Does this need to follow the react version?

Reply:
No, we use the version of the dependency when publishing and don't keep track of that in the package.json version field.
See: https://github.com/finn-no/eik-shared-libs/blob/b400d7ea839e8132af7ea4fc384e6482190bdf9a/scripts/publish.js#L23

Reply:
Noticed an issue though. Because esm-bundle published -fix.1 prerelease versions, we are going to end up publishing react 17.0.2-fix.1 instead of 17.0.2 and in this case we don't want that.